src/main/java/ecdar/controllers/EdgeController.java

```diff
@@ -595,8 +595,9 @@ private void addEdgePropertyRow(final DropDownMenu dropDownMenu, final String ro
     }
 
     private Nail getNewNailBasedOnDropdownPosition() {
-        final double nailX = Math.round(DropDownMenu.x / GRID_SIZE) * GRID_SIZE;
-        final double nailY = Math.round(DropDownMenu.y / GRID_SIZE) * GRID_SIZE;
+        final double nailX = Math.round(DropDownMenu.x / EcdarController.getActiveCanvasZoomFactor().get() / GRID_SIZE) * GRID_SIZE;
+        final double nailY = Math.round(DropDownMenu.y / EcdarController.getActiveCanvasZoomFactor().get() / GRID_SIZE) * GRID_SIZE;
+        System.out.println(EcdarController.getActiveCanvasZoomFactor().get());
```

Comment:
Do we need to have this print line in the final code?

Reply:
:safety_vest: